phoenix-core/src/it/java/org/apache/phoenix/end2end/index/PartialIndexIT.java

```diff
@@ -1053,4 +1056,34 @@ public void testPartialIndexWithIndexHint() throws Exception {
             assertFalse(rs.next());
         }
     }
+
+    @Test
+    public void testPartialIndexOnTableWithCaseSensitiveColumns() throws Exception {
+        try(Connection conn = DriverManager.getConnection(getUrl());
+            Statement stmt = conn.createStatement()) {
+            String dataTableName = generateUniqueName();
+            String indexName = generateUniqueName();
+
+            stmt.execute("CREATE TABLE " + dataTableName
+                    + " (\"hashKeY\" VARCHAR NOT NULL PRIMARY KEY, v1 VARCHAR, \"CoL\" VARCHAR, v3 VARCHAR)");
+
+            stmt.execute("CREATE INDEX " + indexName
+                    + " ON " + dataTableName + "(v1) INCLUDE (\"CoL\") WHERE v3 IS NOT NULL");
```

Comment:
as v3 is in WHERE clause, can we also rename it to something like `coLUmn3`? combination of cases?

Reply:
@virajjasani done